routers/api/packages/generic/generic.go

```diff
@@ -152,15 +150,15 @@ func DeletePackage(ctx *context.Context) {
 func sanitizeParameters(ctx *context.Context) (string, string, string, error) {
 	packageName := ctx.Params("packagename")
 	filename := ctx.Params("filename")
+	packageVersion := ctx.Params("packageversion")
 
 	if !packageNameRegex.MatchString(packageName) || !filenameRegex.MatchString(filename) {
 		return "", "", "", errors.New("Invalid package name or filename")
 	}
 
-	v, err := version.NewSemver(ctx.Params("packageversion"))
-	if err != nil {
-		return "", "", "", err
+	if packageVersion == "" {
```

Comment:
```suggestion
	if strings.TrimSpace(packageVersion) == "" {
```
I would disallow whitespaces.